### DISCUSSION TITLE
More layer height settings

### DISCUSSION THREAD

**Discussion by ripsmooth** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#5] [Ideas]
Hello! Config-file works great but i have one suggestion/question (i am noob with 3d-printing so this might be stypid question).

Could we have more layer height-settings for the config file? Some kind of "High quality/low (and fast) quality" settings would be great for novise users.

Or can i easily change the layer height? I tried to change 0,2mm to 0,1mm (high quality in Creality Print), but it totally messed my test print.

**Comment by suchmememanyskill** [comment]
Changing the layer height should be fine; i'll take a look at this when i have time (probably this weekend).

Do note that the first layer should probably stay at 0.2mm

Also messed up in what way? Kinda curious

**Reply by ripsmooth** [reply]
Good point - I´m not 100% sure did it was 0.1mm or 0.2mm. I think, it was 0.1mm. I already deleted the my version of config, so i cant check it anymore (so that I don't mistakely use it again :D )

Maybe i will try 0,1mm level height / 0,2mm 1st level height, and check how it works!

**Comment by ripsmooth** [comment]
(Sorry my not-so-great english btw)
The mess was: I cancelled the work pretty fast after start when i saw the results. The first layer looked awful, and it was totally low quality. Hard to even say how it looked (sadly i didnt took any photos)

I already did small damages (or i would say tattoos) to bed. The base print got stuck extremely hard to bed, and after big fight to get it off, it let the printed items "tattoo" contours signs to bed.

Maybe the nozzle was too close to bed, and it "engraved" the bed a bit? 

Btw: I read that Ender 3 printers have some kind of magic numbers to set layer height ? 
So should 0.10mm layer height really be to set 0.12mm or 0.16mm in these machines?

**Reply by suchmememanyskill** [reply]
Ya, this is why you leave the first layer height at default, and only change the layer height for other layers. It leaves much less margin for error if you lower the first layer height.

(Ps: i have not had the chance yet to print anything yet. Will update you when i do)

**Comment by suchmememanyskill** [comment]
I have added more layer height profiles in the latest commit. These seem to print ok

**Reply by ripsmooth** [reply]
Super great! I gonna test them, thanks a lot :)

**Reply by suchmememanyskill** [reply]
@ripsmooth kinda curious, what are your results?
